nupic/__init__.py

```diff
@@ -19,3 +19,7 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 __version__ = "0.3.0.dev0"
+
+import os
+NUPIC_ROOT = os.environ.get('NUPIC', os.path.dirname(os.path.realpath(__file__)))
```

Comment:
You guys are right. So I'm favor in keep $NUPIC and in case of it is not set the absolute path is taken..